sdk/lib/drivers/wdf/CMakeLists.txt

```diff
@@ -186,9 +184,13 @@ list(APPEND SOURCE_KM
 # add_registry_inf(wdf01000_reg.inf)
 
 add_library(wdf01000 ${SOURCE_SHARED} ${SOURCE_KM})
+add_ntddi_defines(wdf01000 win8.1 PUBLIC)
+
+# Any component linking against wdf01000 must also target win8.1.
+add_ntddi_defines(wdf01000 win8.1)
```

Comment:
Why calling the function two times?

Reply:
A mistake. Since removed.

Reply:
Appears to have been a mistake. Removed.